src/js/components/Anchor/doc.js

```diff
@@ -2,6 +2,52 @@ import { describe, PropTypes } from 'react-desc';
 
 import { genericProps, getAvailableAtBadge } from '../../utils';
 
+export const themeDoc = {
+  'global.focus.border.color': {
+    description: 'The color around the Anchor when in focus.',
+    type: 'string | { dark: string, light: string }',
+    defaultValue: '#FD6FFF',
+  },
+  'anchor.color': {
+    description: 'Used to apply a default color to the icon and label.',
```

Comment:
I don't think the word 'default' is needed here. I also think starting with "Used to apply" isn't consistent with the other descriptions. Instead, how about something like "The color of the label text and icon strokes"?